src/scr_param.c

```diff
@@ -153,6 +161,16 @@ const char* scr_param_get(const char* name)
 {
   char* value = NULL;
 
+  /* see if this parameter is one which has been set by the application */
+  value = kvtree_elem_get_first_val(scr_app_hash, name);
+  if (value != NULL) {
+    /* evaluate environment variables */
+    if(*value == '$'){
```

Comment:
In case there is a variable embedded anywhere in the value, like "STORE=/tmp/$USER"
```suggestion
    if(strchr(value, '$')){
```

Reply:
Done. I thought I had got all of those when re-doing things to support expandenv.